sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj

```diff
@@ -6,12 +6,13 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.CosmosDB</PackageId>
     <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
-    <Version>1.4.1-preview</Version>
+    <Version>2.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.CosmosDB</AssemblyName>
     <PackageTags>management;cosmosdb;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        Fixing error propagation for APIs concerning Data Plane RBAC and Point in Time Restore feature.
+        Updating with the apiVersion 2020-01-15 azure-rest-api-specs.
+        This patch includes Network Acl Bypass support and Managed Service Identity.
```

Comment:
MAke this more elaborate, cover all 4 new features being introduced.

Reply:
Done, thanks!